src/chapters/06-agents.adoc

```diff
@@ -79,11 +81,11 @@ build once, fingerprint and then reuse
 * Challenge: workspace is *NOT* shared across agents.
 ** A new workspace is allocated for *each* stage
 
-* Pipeline can "pass" artifacts from stage to stage
+* Pipeline can move artifacts from stage to stage
```

Comment:
Semantically, the use of "move" is wrong. It is a copy.
Especially if you want to unstash multiple times, it will lead the end user to confusion.

WDYT about `Pipeline can reuse artifacts...` or `Pipeline can store artifacts...` ?

Reply:
then I would prefer to use "copy", I find it more clear on what's happening

Reply:
👍